### ISSUE TITLE
.gzip extension 

### ISSUE THREAD

**Issue by william0901** [logstash-plugins/logstash-input-s3#62]
logstash-input-s3 currently supports .gz for gzip extension. It makes sense to support .gzip as well.

Will below change work?

https://github.com/logstash-plugins/logstash-input-s3/blob/master/lib/logstash/inputs/s3.rb

from

```
  private
  def gzip?(filename)
    filename.end_with?('.gz')
  end
```

to 

```
  private
  def gzip?(filename)
    filename.end_with?('.gz’) || filename.end_with?(‘gzip’)
  end
```

**Comment by denmak** [comment]
Hi 
It would be greate to fix that, because now I have to work this issue  around   by copying files  from one bucket to another, bacause source is third party bucket and I have no permissions to change extentions. 
many thanks

**Comment by william0901** [comment]
Here is the pull request

https://github.com/logstash-plugins/logstash-input-s3/pull/113

**Comment by yaauie** [comment]
support for the gzip extension was recently merged and released with v3.2.0 of `logstash-input-s3`:
~~~
bin/logstash-plugins update logstash-input-s3
~~~
